### DISCUSSION TITLE
Export BasePlugin and EventEmitter classes

### DISCUSSION THREAD

**Discussion by treio** [katspaugh/wavesurfer.js#2813] [Ideas]
Would it be possible to include **BasePlugin** and **EventEmitter** classes in the export?

I am trying to create my custom plugin, and these two classes would be handy.

Adding these into `package.json` wouldn't be a problem, right?

```
"exports": {
    ....
    "./dist/base-plugin": {
      "import": "./dist/base-plugin.js",
      "require": "./dist/base-plugin.js"
    },
    "./dist/event-emitter": {
      "import": "./dist/event-emitter.js",
      "require": "./dist/event-emitter.js"
    }
  },
```

**Comment by katspaugh** [comment]
Sure, good point! We could export the whole dist I guess.

PR: https://github.com/katspaugh/wavesurfer.js/pull/2821
